map/map_tf_generator/include/map_tf_generator/uniform_random.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Tier IV, Inc.
```

Comment:
```suggestion
// Copyright 2022 TIER IV, Inc.
```

Reply:
Resolved in https://github.com/autowarefoundation/autoware.universe/pull/890/commits/cbaf5fadfcff09f934fff530ce2b1b1893ed29c5